### PR TITLE
Platform Dashboards

### LINKED ISSUE #8169: Add platform dashboards
Add new platform focused dashboards. Also add the `Platform` label to the Any Stat (Prometheus) dashboard so that it can be linked to other platform dashboards.

### PR DESCRIPTION
**Pull request by poulok**
**Description**:
A set of platform oriented dashboards for monitoring the health of networks and diagnosing problems.

**Related issue(s)**:

Fixes #8169 

**Notes for reviewer**:
[Link](https://production.grafana.hedera-ops.com/d/ec9413c6-6f17-49fc-951b-51cbc21a1eef/network-health?from=now-6h&to=now&var-Datasource=grafanacloud-prom&var-network=mainnet&var-name=app_nftsUsed&var-metric_type=All&orgId=1&var-Environment=mainnet&var-NodeID=All) to Network Health dashboard (has links to all other boards)

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

**Review by cody-littley — COMMENTED** [review]
I'd suggest updating the following green/yellow/red thresholds:

- healthyNetworkFraction: move red threshold to 66.667%, this is the number we really care about
- cEvents/sec: red -> 5/sec, yellow -> 10/sec
- stateSignaturesGathered/sec and totalNeverSignedStates/sec shows all red
- rounds/sec: red -> 0.5, yellow -> 0.75
- tipsetAdvancemetn: red -> 0.10, yellow -> 0.15
- bullyScore (name changes in 0.42): yellow -> 2, red -> 5
- dupEv%: red -> 95% We can leave current performance in the yellow band. I'm not happy with it, but we can currently live with it, so red is wrong.)


When I click on one of the links to look at a different page, the buttons reorder themselves. This is because you don't have a button for the current page, so each navigation adds one button and removes another.

What if instead, you had a button that links to each page on each page, but greyed out if you are currently on that page. That way the UI feels more continuous between pages.

**Review by cody-littley — DISMISSED** [review]

**Review by edward-swirldslabs — COMMENTED** [review]
The Conns panel looks like a monotonically increasing value.    That doesn't seem as relevant as seeing the spikes when there is a thrashing network connection.   Should it be an accumulator that is wiped when accessed?

**Review by cody-littley — APPROVED** [review]

File: hedera-node/infrastructure/grafana/dashboards/production/hedera-node/platform/gossip.json
Comment (nathanklick) [inline hedera-node/infrastructure/grafana/dashboards/production/hedera-node/platform/gossip.json:1]: The entire `platform` folder needs to be moved up a level to be under the `production` folder.

**Review by nathanklick — CHANGES_REQUESTED** [review]

File: hedera-node/infrastructure/grafana/dashboards/production/hedera-node/platform/gossip.json
Comment (poulok) [reply hedera-node/infrastructure/grafana/dashboards/production/hedera-node/platform/gossip.json:1]: done

**Review by nathanklick — DISMISSED** [review]

File: hedera-node/infrastructure/grafana/dashboards/production/hedera-node/arbitrary-stat-prom.json [hunk 1]
```diff
@@ -197,7 +212,7 @@
           "refId": "StandardVariableQuery"
         },
         "refresh": 1,
-        "regex": "/^(Reconnect.*|app.*|internal.*|virtual.*|platform.*|merkle.*)/",
+        "regex": "/^(Reconnect.*|app.*|internal.*|virtual.*|internal.*|platform.*)/",
```
Comment (beeradb) [inline hedera-node/infrastructure/grafana/dashboards/production/hedera-node/arbitrary-stat-prom.json:215]:
```suggestion
        "regex": "/^(Reconnect.*|app.*|internal.*|virtual.*|platform.*|merkle.*)/",
```

This was added in https://github.com/hashgraph/hedera-services/pull/8682/files to allow `merkle_*` metrics to show up in this dashboard.

**Review by beeradb — APPROVED** [review]

**Pull request merged**